pkg/runtime/service_controller.go

```diff
@@ -187,13 +189,39 @@ func (c *serviceController) BindControllerManager(mgr ctrlrt.Manager, cfg ackcfg
 	c.metaLock.Lock()
 	defer c.metaLock.Unlock()
 
-	cache := ackrtcache.New(c.log)
-	if cfg.WatchNamespace == "" {
+	namespaces, err := cfg.GetWatchNamespaces()
+	if err != nil {
+		return fmt.Errorf("unable to get watch namespaces: %v", err)
+	}
+
+	cache := ackrtcache.New(c.log, ackrtcache.Config{
+		WatchScope: namespaces,
+		// Default to ignoring the ack-system kube-system, kube-public, and
+		// kube-node-lease namespaces.
+		// NOTE: Maybe we should make this configurable? It's not clear that
+		// we'd ever want to watch these namespaces.
+		Ignored: []string{
+			"ack-system",
```

Comment:
Do we really want to go down the path of hiding what is ignored? It gets into a slippery slope. Maybe for some `kube` namespaces, but I don't think `ack-system` should be ignored. There are probably users out there that are creating CR's in this namespace since it's the default namespace that the controller runs in.

Reply:
Since we added cross-account resource management to ACK, the caches ignored to the `ack-system` namespace (or the namespace where the controller is running). Still, the controller will handle resources in that namespace using the controller role. This means that only CARM will not work on `ack-system`. 

Now, looking at the bigger picture, I agree it might be a bit odd, but we might just allow it since there don't seem to be any issues.. and it won't have any side effects

Reply:
I still don't think it's a good idea to expose this level of configuration at the helm/flag level of controllers. However i'm down to remove `ack-system` from the ignored list and only leave `kube-system`, `kube-public` and `kube-node-lease`.

Reply:
> I still don't think it's a good idea to expose this level of configuration at the helm/flag level of controllers. However i'm down to remove `ack-system` from the ignored list and only leave `kube-system`, `kube-public` and `kube-node-lease`.

I also do not thing adding a configuration for this is needed, that would add more work/confusion and we'd need to change some documentation.